scripts/tidb.json

```diff
@@ -11088,16 +11287,16 @@
       "type": "row"
     }
   ],
-  "refresh": "30s",
+  "refresh": false,
   "schemaVersion": 18,
   "style": "dark",
   "tags": [],
   "templating": {
     "list": []
   },
   "time": {
-    "from": "now-1h",
-    "to": "now"
+    "from": "2020-01-02T16:10:11.043Z",
```

Comment:
Why change this?

Reply:
I find that if I do some operation and click export, the content of the json file would be different.
This time it should be correct.

Reply:
OK